frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkLegaleseModal/EmbeddingSdkLegaleseModal.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import {
+  Button,
+  Group,
+  List,
+  Modal,
+  type ModalProps,
+  Text,
+  Title,
+} from "metabase/ui";
+
+import type { AdminSettingComponentProps } from "../types";
+
+export const EmbeddingSdkLegaleseModal = ({
+  opened,
+  onClose,
+  updateSetting,
+}: AdminSettingComponentProps & ModalProps) => {
+  const [loading, setLoading] = useState(false);
+
+  const onAccept = async () => {
+    setLoading(true);
+    await Promise.all([
+      updateSetting({ key: "show-sdk-embed-terms" }, false),
+      updateSetting({ key: "enable-embedding-sdk" }, true),
+    ]);
+    setLoading(false);
```

Comment:
Should we also do `setLoading(false) + onClose()` (and optionally show an error) if the updateSetting promise fails for some reason too?